docs/static/dead-letter-queues.asciidoc

```diff
@@ -125,6 +125,26 @@ Use the `dead_letter_queue.storage_policy` option to control which entries shoul
 Set the value to `drop_newer` (default) to stop accepting new values that would push the file size over the limit. 
 Set the value to `drop_older` to remove the oldest events to make space for new ones.
 
+===== Automatic cleaning of consumed events
+
+By default, dead letter queue input plugin doesn't remove the events that it consumes, it just commits a reference
+to avoid re-processing.
```

Comment:
```suggestion
By default, the dead letter queue input plugin does not remove the events that it consumes.
Instead, it commits a reference to avoid re-processing events.
```